v0-991/learn/tools-ides/intellij-plugin/using-the-intellij-plugin.md

```diff
@@ -17,39 +20,39 @@ Follow the steps below to create a new Ballerina project.
 1. Open IntelliJ, click **File** in the top menu, click **New**, and then click **Project**.
 
 2. Select **Ballerina** as the type of the project, and click **Next**.
-![Install the plugin via IntelliJ IDEA](../images/select-project-type.png)
+![Install the plugin via IntelliJ IDEA](/v0-991/learn/images/select-project-type.png)
 
 3. Select a Ballerina SDK for the project, and click **Next**.
 
     >**Tip:** **If you do not have an already-configured Ballerina SDK to select,** click **Configure**, select the location of the Ballerina distribution, click **Open**, and then click **Next** to continue with the project creation.
 
-    ![Select a Ballerina SDK](../images/select-sdk.png)
+    ![Select a Ballerina SDK](/v0-991/learn/images/select-sdk.png)
    
 4. Enter a name for the project, a location to save it, and click **Finish**.
 
-    ![Enter project name and location](../images/enter-project-name-and-location.png)
+    ![Enter project name and location](/v0-991/learn./images/enter-project-name-and-location.png)
```

Comment:
there is a additional `.` in the url .... have to remove it.

Reply:
Removed.